en/v3.0.0/resources/example-behaviour.js

```diff
@@ -1,4 +1,47 @@
 ;(function() {
+
+  // show outdated version warning
+  var container = document.getElementsByClassName('container-fluid')[0];
```

Comment:
Our main template has an element with `id="title"`.  Would it make sense to insert the warning before that instead?  Not sure that is more or less reliable.  So no need to change if you prefer not to.